modules/saml/www/sp/saml2-acs.php

```diff
@@ -136,15 +141,16 @@
     // the status of the response wasn't "success"
     $e = $e->toException();
     \SimpleSAML\Auth\State::throwException($state, $e);
+    return;
```

Comment:
This should never be necessary. I'd suggest that instead of this, we fix `throwException()` so that even if the exception handler function returns (which it shouldn't), the method doesn't return itself. Maybe raise an exception warning about a bogus exception handler?

Reply:
Done